### DISCUSSION TITLE
Roborock cloud?

### DISCUSSION THREAD

**Discussion by done7k** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#9] [Q&A] [answered]
Is there any hope for a similar tool for Roborock accounts?

**Comment by PiotrMachowski** [comment]
Is there any benefit of using Roborock instead of Xiaomi Home?

**Reply by done7k** [reply]
I tried MiHome but I found Roborock more stable on iPad. I see others prefer Roborock as well https://www.reddit.com/r/Roborock/comments/gbd0g2/roborock_app_or_mi_home_app_for_s6_may_2020/

**Reply by askusbloodfist** [reply]
The S6 Maxv Get allso the Remote Camera funktion that is still not in the Mi home. 
And you allso get just the vacume and no extra that some prefer. Why they leave out the camera funktion is wierd. But consider that its still not come sins released in Juli last year. Id say it safe to say it might not come to Mi home at all.

**Reply by jappish84** [reply]
Main reason for me to use the roborock app token instead would be because the vac works with Google assistant and specific room cleaning commands, out of the box. This would enable one to use both home assistant and Google assistant to control the vac when wanting to clean specific rooms.

I know one can setup room/zone cleaning in home assistant now, and control it with home assistant but it's pretty hacky with calibrating zone coordinates etc

**Reply by PiotrMachowski** [reply]
@jappish84 you don't have to calibrate zone coordinates, you can use room numbers.

**Reply by jappish84** [reply]
Oh, didn't know that, just saw some post about it over at home assistant, thanks! 

It doesn't seem to work with Google assistant though (without having to try and trick GA with some clever naming) or am I missing something?

**Reply by PiotrMachowski** [reply]
I'm not sure, I don't use it, but as far as I know you can use scripts to do it

**Reply by FluffyGhoster** [reply]
Seems like a lot of people uses it, it's the new shiny version and it has features the other has not (like the live feed), a lot more people use it too according to the pools like https://www.reddit.com/r/Roborock/comments/rg5ra3/do_you_use_roborock_or_mi_home_app/

I think it would be a nice step forward to have it for HA, I use it myself and would love to have it working in HA

**Comment by PiotrMachowski** [comment, marked as answer]
Roborock accounts use Tuya, it's a completely different project
